lib/TPP/TileConsumerAndFuseProducers.cpp

```diff
@@ -140,10 +272,30 @@ struct TileConsumerAndFuseProducers
     linalg::registerTilingInterfaceExternalModels(registry);
   }
   void runOnOperation() override {
+    func::FuncOp func = getOperation();
+    transform::TrivialPatternRewriter rewriter(&getContext());
+    llvm::SmallDenseSet<Operation *> fusedOps;
+    func->walk([&](linalg::LinalgOp linalgOp) {
+      if (linalg::isElementwise(linalgOp) && hasMatmulLikeProducer(linalgOp)) {
+        SmallVector<int64_t> actualTileSizes = llvm::to_vector(this->tileSizes);
+        // Default tiling scheme tile by 32.
+        if (actualTileSizes.empty())
```

Comment:
You merged without addressing this issue. Please create a new issue to change this behaviour.

Reply:
You want to move the default in the constructor?

Reply:
the default of {32,32} is specific for this pattern.

Reply:
No, the tile size of 32 is a "good enough" default for any pattern, because it's what libxsmm prefers to run. Anything smaller than this, by default, should NOT be tiled (unless forced via this option).